UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/Data/Reactions/FreonFireReaction.cs

```diff
@@ -50,7 +50,7 @@ public void React(GasMix gasMix, MetaDataNode node)
 
 					gasMix.RemoveGas(Gas.Freon, freonBurnRate);
 					gasMix.RemoveGas(Gas.Oxygen, freonBurnRate * oxygenBurnRate);
-
+					gasMix.RemoveGas(Gas.Plasma, freonBurnRate);
```

Comment:
I very much object to this change honestly
While I understand the intent to make the freon event not have the station burn down, this isnt really going to do much to solve that and the change to the temperature range (which as a side note, if we want to mirror tg should be 120k to 160k) should fix this

Reply:
Though what does it change in terms of making though?
I don't feel like it has that much of a big impact, if you're going out of your way to make it

Reply:
well, freonfire handles the freon cooling effect, and it does so in the proper way, requiring oxygen to cool and producing co2 as a byproduct whilst also slowly cooling it over time, so you are effectively gating the proper cooling method behind the presence of plasma

Id personally say removing this change (and the change to the energy release part) in addition to removing the improper freon cooling implementation would be nice, but yknow

Reply:
the problem is it gets too cold for gameplay, I picked a temperature that winter coats can protect you from, and is not to devastating to be in without protection, making it simulated  base will make it really hard to balance

Reply:
will work on your PR to add the changes required to balance numbers around but would be good+